lib/json.go

```diff
@@ -79,131 +76,89 @@ import (
 //
 //	'{"a":1}{"b":2}'.decode_json_stream()   // return [{"a":1}, {"b":2}]
 //	b'{"a":1}{"b":2}'.decode_json_stream()  // return [{"a":1}, {"b":2}]
-func JSON(adapter ref.TypeAdapter) cel.EnvOption {
+func JSON(adapter types.Adapter) cel.EnvOption {
```

Comment:
This is not a breaking change as `types.Adapter` is [`type Adapter = ref.TypeAdapter`](https://pkg.go.dev/github.com/google/cel-go@v0.20.1/common/types#Adapter).